LogicalTypes.md

```diff
@@ -366,3 +366,7 @@ optional group my_map (MAP_KEY_VALUE) {
 }
 ```
 
+## Null
+Sometimes when discovering the schema of existing data values are always null and there's no type information.
```

Comment:
We'll have to make sure this works. I think our Schema union logic isn't currently based on logical types. For example, DECIMAL backed by binary merged with one backed by int64 probably fails. Probably not a big deal.